### PR TITLE
Roman alter table

### PR DESCRIPTION
**Pull request by romanbracinik**
[BIG-36](https://keboola.atlassian.net/browse/BIG-36)

Table utils: https://github.com/keboola/php-table-backend-utils/pull/88
Common: https://github.com/keboola/php-storage-driver-common/pull/21
TD Driver: https://github.com/keboola/php-storage-driver-teradata/pull/49
BQ Driver: https://github.com/keboola/php-storage-driver-bigquery/pull/27

### CODE REVIEW

File: src/Handler/Table/Alter/DropColumnHandler.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        assert($command instanceof DropColumnCommand);
+
+        assert($command->getPath()->count() === 1, 'TableColumnShared.path is required and size must equal 1');
+        assert($command->getTableName() !== '', 'TableColumnShared.tableName is required');
+
+        $bqClient = $this->clientManager->getBigQueryClient($credentials);
+
+        // define columns
+        // validate
+        assert($command->getColumnName() !== '', 'DropColumnCommand.name is required');
```
Comment (jirkasemmler) [inline src/Handler/Table/Alter/DropColumnHandler.php]:
```suggestion
        assert($command->getColumnName() !== '', 'DropColumnCommand.columnName is required');
```

File: src/Handler/Table/Alter/DropColumnHandler.php [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+     */
+    public function __invoke(
+        Message $credentials,
+        Message $command,
+        array $features
+    ): ?Message {
+        assert($credentials instanceof GenericBackendCredentials);
+        assert($command instanceof DropColumnCommand);
+
+        assert($command->getPath()->count() === 1, 'TableColumnShared.path is required and size must equal 1');
```
Comment (jirkasemmler) [inline src/Handler/Table/Alter/DropColumnHandler.php]:
```suggestion
        assert($command->getPath()->count() === 1, 'DropColumnCommand.path is required and size must equal 1');
```

File: src/Handler/Table/Alter/DropColumnHandler.php [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        assert($command->getTableName() !== '', 'TableColumnShared.tableName is required');
```
Comment (jirkasemmler) [inline src/Handler/Table/Alter/DropColumnHandler.php]:
```suggestion
        assert($command->getTableName() !== '', 'DropColumnCommand.tableName is required');
```

File: src/Handler/Table/Alter/AddColumnHandler.php [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+        Message $command,
+        array $features
+    ): ?Message {
+        assert($credentials instanceof GenericBackendCredentials);
+        assert($command instanceof AddColumnCommand);
+
+        $column = $command->getColumnDefinition();
+        // validate
+        assert($command->getPath()->count() === 1, 'TableColumnShared.path is required and size must equal 1');
+        assert($command->getTableName() !== '', 'TableColumnShared.tableName is required');
```
Comment (jirkasemmler) [inline src/Handler/Table/Alter/AddColumnHandler.php]:
```suggestion
        assert($command->getTableName() !== '', 'AddColumnCommand.tableName is required');
```

File: src/Handler/Table/Alter/AddColumnHandler.php [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+        Message $credentials,
+        Message $command,
+        array $features
+    ): ?Message {
+        assert($credentials instanceof GenericBackendCredentials);
+        assert($command instanceof AddColumnCommand);
+
+        $column = $command->getColumnDefinition();
+        // validate
+        assert($command->getPath()->count() === 1, 'TableColumnShared.path is required and size must equal 1');
```
Comment (jirkasemmler) [inline src/Handler/Table/Alter/AddColumnHandler.php]:
```suggestion
        assert($command->getPath()->count() === 1, 'AddColumnCommand.path is required and size must equal 1');
```

**Review by jirkasemmler — CHANGES_REQUESTED** [review]
jen ty nazvy, mam spatne i u sebe

**Comment by romanbracinik** [comment]
Dik upravil som

**Review by jirkasemmler — APPROVED** [review]
cool!

**Pull request merged**